akvo/iati/imports/mappers/ICCO/descriptions.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+
+# Akvo Reporting is covered by the GNU Affero General Public License.
+# See more details in the license.txt file located at the root folder of the Akvo RSR module.
+# For additional details on the GNU license please see < http://www.gnu.org/licenses/agpl.html >.
+
+from ..descriptions import Descriptions
+
+description_start = dict(
+    project_name='project name:',
+    subtitle='subtitle:',
+    project_plan_summary='project summary:',
+    background='background:',
+    current_status='baseline situation:',
+    project_plan='project plan:',
+    sustainability='sustainability:',
+)
+
+def _text_starts_with(text, start):
+    return text.lower().startswith(start)
+
+class Descriptions(Descriptions):
```

Comment:
No it *has to* have the same name. The reason for this is that IatiImport.model_mappers() is a kind of class factory method that loops over all names in iati.imports.mappers.\_\_all\_\_ and imports that name, after fist having checked if the import should be done from mappers or a named submodule of mappers like mappers.ICCO. So what should be renamed is IatiImport.mapper_prefix field since it's not a prefix any more. I started out with the idea that subclasses would be prefixed with the partner's name, but that didn't work out in practice.